test/e2e_test.sh

```diff
@@ -19,7 +19,7 @@ set -ex
 echo "setting up OIDC provider"
 pushd ./test/fakeoidc
 oidcimg=$(ko build main.go --local)
-docker network ls | grep fulcio_default || docker network create fulcio_default
+docker network ls | grep fulcio_default || docker network create --label com.docker.compose.network=fulcio_default fulcio_default
```

Comment:
This works for me if I have

```suggestion
docker network ls | grep fulcio_default || docker network create --label com.docker.compose.network=default fulcio_default
```